apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/TraceHeaderContext.cs

```diff
@@ -50,13 +50,13 @@ internal sealed class TraceHeaderContext
         /// <summary>Gets the span id or null if none is available.</summary>
         public ulong? SpanId { get; }
 
-        /// <summary>True if the request should be traced.</summary>
-        public bool ShouldTrace { get; }
+        /// <summary>True if the request should be traced or null if it is not available.</summary>
```

Comment:
This is a little confusing. If what is not available? Perhaps "True if the request should be traced. Null if the trace header did not indicate whether it should be traced or not."